custom/icds_reports/migrations/sql_templates/database_functions/aggregate_awc_data.sql

```diff
@@ -501,6 +503,12 @@ BEGIN
   EXECUTE 'UPDATE ' || quote_ident(_tablename5) || ' agg_awc SET ' ||
     'num_awc_infra_last_update = 0 WHERE infra_last_update_date IS NULL';
 
+  EXECUTE 'UPDATE ' || quote_ident(_tablename5) || ' agg_awc SET ' ||
+    'num_awc_infra_last_updated_in_half_of_a_year = 1 WHERE infra_last_update_date IS NOT NULL AND ' || quote_literal(_month_start_6m) || ' < infra_last_update_date';
+
+  EXECUTE 'UPDATE ' || quote_ident(_tablename5) || ' agg_awc SET ' ||
+    'num_awc_infra_last_updated_in_half_of_a_year = 0 WHERE infra_last_update_date IS NULL OR ' || quote_literal(_month_start_6m) || ' >= infra_last_update_date';
```

Comment:
Can you combine this update with the previous 3? I think it could look something like

```
UPDATE table SET
   num_awc_infra_last_update = CASE WHEN infra_last_update_date IS NULL THEN 0 ELSE 1 END,
   num_awc_infra_last_updated_in_half_of_a_year = CASE WHEN infra_last_update_date < 6 months ago THEN 1 ELSE 0 END
```